packages/run-protocol/src/vaultFactory/types.js

```diff
@@ -37,24 +37,22 @@
  */
 
 /**
- * @typedef  {Object} VaultFactory - the creator facet
+ * @typedef {Object} VaultFactory The creator facet
  * @property {AddVaultType} addVaultType
- * @property {() => Promise<Array<Collateral>>} getCollaterals
+ * @property {() => Promise<Collateral[]>} getCollaterals
  * @property {() => Allocation} getRewardAllocation,
  * @property {() => Instance} getContractGovernor
  * @property {() => Promise<Invitation>} makeCollectFeesInvitation
  */
 
 /**
- * @callback ReallocateWithFee
- *
- * Transfer the indicated amount to the vaultFactory's reward
- * pool, taken from the `fromSeat`. Then reallocate over all the seat
- * arguments and the rewardPoolSeat.
+ * Transfer the indicated amount to the vaultFactory's reward pool, taken from
+ * the `fromSeat`. Then reallocate over all the seat arguments and the rewardPoolSeat.
  *
+ * @callback ReallocateWithFee
```

Comment:
Did it swap the declaration and prose automatically, or did manually intervene? Do they both mean the same thing in jsdoc?